wcfsetup/install/files/lib/acp/form/ArticleAddForm.class.php

```diff
@@ -645,6 +646,23 @@ public function assignVariables()
 
         SmileyCache::getInstance()->assignVariables();
 
+        // readData not work in edit form
```

Comment:
What does this mean?

Reply:
It was just a comment to me that I can't implement the code below in the readData function. Because in the ArticleEditForm the content is changed with the content from the edit object, after this code will be executed.